script.js

```diff
@@ -49,4 +49,90 @@ const photos = [
     },
 ];
 
-console.log('hello');
\ No newline at end of file
+const section = document.getElementById('gallery');
+const more_photos = document.getElementById('loadMore');
+// filter buttons
+const allButton = document.getElementById('all');
+const cityButton = document.getElementById('city');
+const natureButton = document.getElementById('nature');
+const animalButton = document.getElementById('animals');
+//filer classes
+const all_photos = document.getElementsByClassName('all');
```

Comment:
Hi Rochelle, you may want to test out your page again. When you click the "All" and "Nature" filters, it works correctly, but when you click the "City" and "Animals" filters, it's either not showing all the correct images or it's not showing any images at all. You may want to take a look where you are creating all the arrays for photo type, because something may be going wrong there.

Reply:
I don't why it's not working on your end because it's working on mine, but I changed the photo filter function to use a switch statement like you said in the other comment, so hopefully it works now.